src/i18n/strings/en_EN.json

```diff
@@ -502,6 +502,7 @@
     "Collecting app version information": "Collecting app version information",
     "Collecting logs": "Collecting logs",
     "Uploading report": "Uploading report",
+    "Downloading report": "Downloading report",
```

Comment:
It seems odd for some strings to say "logs" and other to say "report"... "Logs" seems more prevalent, perhaps change the "uploading / downloading report" strings to "logs" instead?

Reply:
So for `uploading report` the action button is `Send report` so that makes sense
Could change it for `downloading report` if we wanted to

Reply:
I'd advocate for consistency with logs for this specific instance, as it relates to 'Download logs'.